include/revng/Pipeline/LLVMContainer.h

```diff
@@ -46,6 +46,9 @@
 
 namespace pipeline {
 
+void makeGlobalObjectsArray(llvm::Module &Module,
+                            llvm::StringRef GlobalArrayName);
```

Comment:
What's this for? Doc

Reply:
it creates the array with a pointer to each global object so that they can't be erased by the linker.